pom.xml

```diff
@@ -632,7 +632,7 @@
         <spark311cdh.version>3.1.1.3.1.7270.0-253</spark311cdh.version>
         <spark312.version>3.1.2</spark312.version>
         <spark313.version>3.1.3-SNAPSHOT</spark313.version>
-        <spark320.version>3.2.1-SNAPSHOT</spark320.version>
+        <spark320.version>3.2.0-rc3</spark320.version>
```

Comment:
There is no 3.2.0-rc3 in the snapshots repo.

https://repository.apache.org/content/repositories/snapshots/org/apache/spark/spark-core_2.12/